src/controller.cpp

```diff
@@ -59,6 +63,15 @@ namespace bzzz
         motorFR = clip(mFR, motorClipLow, motorClipHigh);
         motorBL = clip(mBL, motorClipLow, motorClipHigh);
         motorBR = clip(mBR, motorClipLow, motorClipHigh);
+
```

Comment:
This is not correct - check these notes: [Hexacopter.pdf (v1)](https://github.com/user-attachments/files/19665506/Hexacopter.pdf)